integration/spark-common-test/src/test/resources/partition_data.csv

```diff
@@ -0,0 +1,27 @@
+id,vin,logdate,phonenumber,country,area,salary
```

Comment:
Oh, this file is copied from example package. Maybe I can reduce them and keep only one.